### PR TITLE
Fix local build and install, originally by jiyongj 

### PR DESCRIPTION
**Pull request by baulus**
The original issue description is here 
https://github.com/opencybersecurityalliance/stix-shifter/pull/620

### CODE REVIEW

**Review by yurii-klymenko — APPROVED** [review]

**Comment by yurii-klymenko** [comment]
You need to update the development documentation @baulus

File: README.md [hunk 1]
```diff
@@ -56,10 +56,26 @@ $ stix-shifter translate qradar query {} "[ipv4-addr:value = '127.0.0.1']" {}
 
 **Note:** In order to build `stix-shifter` packages from source follow the below prerequisite steps:
    1. Go to the stix-shifter parent directory
-   2. Generate latest requirements.txt: `python3 generate_requirements.py`
-   3. Install the dependencies in your python 3 environment: `pip install -r requirements.txt` 
-   4. Alternatively you can create a Python 3 virtual environemnt:
-       `virtualenv -p python3 virtualenv && source virtualenv/bin/activate && pip install -r requirements-dev.txt`
+   2. Run setup: `python3 setup.py install`
+   3. Alternatively you can create a Python 3 virtual environemnt:
+       `virtualenv -p python3 virtualenv && source virtualenv/bin/activate && python3 setup.py install`
```
Comment (mdazam1942) [inline README.md:61]: I don't think you need to keep **Note:** section as you are adding a new subsection `### Running From the Source`

Comment (baulus) [reply README.md:61]: I will remove Note keyword, but the explanation belongs to this section

File: README.md [hunk 2]
```diff
@@ -54,12 +54,30 @@ Example:
...
-   4. Alternatively you can create a Python 3 virtual environemnt:
-       `virtualenv -p python3 virtualenv && source virtualenv/bin/activate && pip install -r requirements-dev.txt`
+   2. Optionally, you can create a Python 3 virtual environemnt:
+       `virtualenv -p python3 virtualenv && source virtualenv/bin/activate`
+   3. Run setup: `python3 setup.py install`
+
+
+### Running From the Source
+
+You may also use `python3 main.py` local script. All the options are the same as for the *"As A Command Line Utility"* usage above.
```
Comment (mdazam1942) [inline README.md:66]: You may also use `python3 main.py` local script. All the options are the same as *"As a command line utility"* usage above.

File: README.md [hunk 3]
```diff
@@ -54,12 +54,30 @@ Example:
...
+python3 main.py translate qradar query {} "[ipv4-addr:value = '127.0.0.1']" {}
+```
+
+In order to run `python3 main.py` from the source follow the below prerequisite steps:
+   1. Go to the stix-shifter parent directory
+   2. Optionally, you can create a Python 3 virtual environemnt:
+       `virtualenv -p python3 virtualenv && source virtualenv/bin/activate`
+   3. Run setup to install dependancies: `INSTALL_REQUIREMENTS_ONLY=1 python3 setup.py install`. 
+
+**Note:** INSTALL_REQUIREMENTS_ONLY=1 is an directive to forse the setup to stop after it installs the pip dependances. This oprtion is similar to `python3 generate_requirements.py && pip install -r requirements.txt`
```
Comment (mdazam1942) [inline README.md:80]: some typo fix-
**Note:** setup.py only installs dependencies when INSTALL_REQUIREMENTS_ONLY=1 directive is used. This option is similar to `python3 generate_requirements.py && pip install -r requirements.txt`

**Review by yurii-klymenko — APPROVED** [review]

**Review by mdazam1942 — APPROVED** [review]

**Pull request merged**